### ISSUE TITLE
Implement new command structure in fsm code

### ISSUE THREAD

**Issue by poornack** [uvic-auvic/Motor_Controller_2.0#10]
There is a new command structure for the motor controller. This needs to be implemented in the fsm code.

**Comment by poornack** [comment]
The FSM code has been updated with the new command structure. The function asciiToInt converts the incoming arguments in ASCII encoding to an integer. I have also added printing out error messages when an incorrect command is entered. This will be helpful when debugging in the future. The way the code is written, only commands that are 100% syntactically correct will get executed. Otherwise, an error will be returned.

I have entered in the comments where function calls to change motor speed, stop motor, etc, should go. When the interface for controlling the motors is completed(in another issue), those functions can be called from the FSM code in the appropriate location.

I have also made a small change to the UART code. Now when the enter key is repeatedly pressed(\n character is sent to the board), the UART code will not wake the FSM task. It won't do anything.

Assigning this to Lyden to review and merge to master.
